openfifth/aspen-discovery
# NEW FEAT: Summon connector

Summon Connector Test Plan

1) Download the patch.
2) When ADB is running, navigate to localhost:8083.
3) Sign in to Aspen as admin.
4) Complete database updates.
5) Use the burger menu to navigate to the administration area.
6) In the admin section, navigate to System Administration and select Modules.
7) Once in Modules, navigate to the Summon option and tick enable, there is no need to change anything else on this page.
8) In the Administration menu, you should now have a summon box.
9) In this box, choose Summon Settings.
10) Click to add new settings. The Summon API is: http://api.summon.serialssolutions.com.
11) The Id and password for the House of Lords are available in PassBolt.
12) Once you have saved the settings, navigate to Primary Configuration and scroll down to Summon. 
13) Using the dropdown, select ‘Summon’ or the name that you gave to your Summon Settings in step 10. 
14) Once you have saved these settings, use the main search bar and set the last dropdown to ‘Articles and Databases’
15) You can now explore Summon.
